app/src/main/java/com/cornellappdev/android/eatery/ui/screens/FavoritesScreen.kt

```diff
@@ -139,118 +139,157 @@ fun FavoritesScreen(
+                            } else {
+                                listOf(
+                                    Filter.CustomFilter.Today,
+                                    Filter.FromEatery.North,
+                                    Filter.FromEatery.West,
+                                    Filter.FromEatery.Central,
+                                )
+
                             }
                         )
                     }
-                    LazyColumn(
-                        modifier = Modifier.fillMaxSize(),
-                        verticalArrangement = Arrangement.spacedBy(12.dp),
-                        horizontalAlignment = Alignment.CenterHorizontally
-                    ) {
-                        if (toggle.value) {
-                            items(
-                                items = favoriteEateries,
-                                key = { eatery ->
-                                    eatery.id!!
-                                }) { eatery ->
-                                EateryCard(
-                                    eatery = eatery,
-                                    isFavorite = true,
-                                    onFavoriteClick = {
-                                        if (!it) {
-                                            favoriteViewModel.removeFavorite(eatery.id)
-                                        }
-                                    }) {
-                                    onEateryClick(it)
-                                }
-                            }
-                        } else {
-                            items(favoritesScreenViewState.favoriteCards) { itemFavoritesCardViewState ->
-                                ItemFavoritesCard(
-                                    itemFavoritesCardViewState
-                                )
+                    if (toggle) {
+                        item {
+                            AnimatedVisibility(favoriteEateries.isEmpty()) {
+                                EateriesEmptyState(message = "You currently have no favorite eateries!")
                             }
                         }
 
+                        items(
+                            items = favoriteEateries,
+                            key = { eatery ->
+                                eatery.id!!
+                            }) { eatery ->
+
+                            EateryCard(
+                                eatery = eatery,
+                                isFavorite = true,
+                                modifier = Modifier.animateItemPlacement(),
+                                onFavoriteClick = {
+                                    if (!it) {
+                                        favoriteViewModel.removeFavorite(eatery.id)
+                                    }
+                                }) {
+                                onEateryClick(it)
+                            }
+                        }
```

Comment:
The indentation here's getting a bit hard to read. Can we extract this into a private composable?

The extract tool veronica showed should do this for you, but if you want it to compile in a lazy column body, you can do `fun LazyColumnScope.[name]()` or something like that, forgot exactly what it looks like